### PR TITLE
fix: introduce Avatar component

### PR DESCRIPTION
**Pull request by InnaAtanasova**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/2830

#### Please provide a brief summary of this pull request.
This PR introduces Avatar component that was released in Fundamental-styles a while ago.
Also adds Deprecation alerts for Identifier and Image components

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/avatar/avatar.component.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import '~fundamental-styles/dist/avatar';
+
+// Temporary style fix until the latest version of Fundamental-styles is released
+fd-avatar[role='button'] {
+    cursor: pointer;
+}
+
+.fd-avatar--accent-color-1 {
```
Comment (JKMarkowski) [inline libs/core/src/lib/avatar/avatar.component.scss:8]: Don't we have them on styles?

File: libs/core/src/lib/avatar/avatar.component.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            this.transparent ? 'fd-avatar--transparent' : '',
+            this.placeholder ? 'fd-avatar--placeholder' : '',
+            this.tile ? 'fd-avatar--tile' : '',
+            this.class
+        ]
+            .filter((x) => x !== '')
+            .join(' ');
+    }
+
+    elementRef(): ElementRef<any> {
```
Comment (JKMarkowski) [inline libs/core/src/lib/avatar/avatar.component.ts:124]: Could you add hidden comment, hiding it from documentation?

File: libs/core/src/lib/avatar/avatar.component.scss (hunk 1 above)
Comment (InnaAtanasova) [reply libs/core/src/lib/avatar/avatar.component.scss:8]: We do, but it's not released yet. The current styles version (in ngx) uses `has-color-`

File: libs/core/src/lib/avatar/avatar.component.ts (hunk 2 above)
Comment (InnaAtanasova) [reply libs/core/src/lib/avatar/avatar.component.ts:124]: Yes, thanks! I missed this one :D

**Pull request merged**